pyproject.toml

```diff
@@ -2,8 +2,11 @@
 name = "las-geoh5"
 version = "0.1.0"
 description = "Las/Geoh5 conversion"
+license = "MIT"
+readme = "README.md"
+homepage = "https://mirageoscience.com"
 authors = ["Mira Geoscience <support@mirageoscience.com>"]
-keywords = ["geology", "geophysics", "earth sciences"]
+keywords = ["packaging", "poetry"]
```

Comment:
previous keywords were goo, no?

Reply:
Thought you asked to change the keywords